pkg/kubelet/kubelet.go

```diff
@@ -2138,9 +2150,15 @@ func (kl *Kubelet) syncLoopIteration(updates <-chan kubetypes.PodUpdate, handler
 			glog.Errorf("Kubelet does not support snapshot update")
 		}
 	case <-kl.resyncTicker.C:
```

Comment:
I agree kubelet.go is too large, but the sync ticker is not remotely concerning to me. We have node status updates, and various volume/network related code in this file. For me, those are on the top of the list to get moved out. In this case, I think it's actually clear to have the ticker in the main sync loop and it also avoids unnecessary goroutine/complexity in the work queue.